health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/Switch.js

```
@@ -0,0 +1,62 @@
+      <div
+        className={`digit-switch ${isChecked ? "checked" : ""} ${shapeOnOff ? "shape-onoff" : ""} ${disable ? "switch-disabled" : ""}`}
+        onClick={handleToggle}
+        tabIndex={0}
+        style={switchStyle || {}}
+        onKeyDown={(e) => {
+          if (e.key === "Enter" || e.key === " ") {
+            handleToggle();
+          }
+        }}
+      >
+        {shapeOnOff && isChecked && <div className="digit-switch-shape-on"></div>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)